### DISCUSSION TITLE
[Enhancement] Add script icon to userscript header

### DISCUSSION THREAD

**Discussion by ghost** [victornpb/undiscord#444] [3. Ideas] [answered]
The line below can be added to the // ==UserScript== area so the logo of the script shows up in our script manages. I suggest using the logo for the project here.

Example from another project:
// @icon https://raw.githubusercontent.com/DenverCoder1/Unedit-for-Reddit/3.9.4/images/logo128.png

As you can see, many other scrips take advantage of this feature, and it would make a good addition.
![](https://user-images.githubusercontent.com/76442625/214338736-2d1e8783-e659-4a0f-989f-c5d8f743b8a8.png)

**Comment by victornpb** [comment]
I have no idea why I haven't added this yet 😂, I thought I've done it but I realized I didn't.
Thank you for reminding me of that.

**Comment by victornpb** [comment, marked as answer]
Implemented on 5.1 https://github.com/victornpb/undiscord/discussions/456
